src/java/test/org/apache/zookeeper/server/ZooKeeperServerStartupTest.java

```diff
@@ -167,6 +167,7 @@ public void testClientConnectionRequestDuringStartupWithNettyServerCnxn()
      */
     @Test(timeout = 30000)
     public void testFourLetterWords() throws Exception {
+        System.setProperty("zookeeper.test.4lw.enabled", "true");
```

Comment:
I is better to use zookeeper.4lw.commands.whitelist. This comment is for all the test classes where zookeeper.test.4lw.enabled used.